packages/actor-init-sparql/lib/ActorInitSparql-browser.ts

```diff
@@ -169,6 +169,32 @@ export class ActorInitSparql extends ActorInit implements IActorInitSparqlArgs {
     // Execute query
     const resolve: IActionQueryOperation = { context, operation };
     const output = await this.mediatorQueryOperation.mediate(resolve);
+    // Set bindings
```

Comment:
I would suggest moving all of this new code into a new static method (`enhanceQueryResults`?). This would also simplify testing.